client/src/app/tabs/bpmn/modeler/BpmnModeler.js

```diff
@@ -17,6 +17,10 @@ import globalClipboardModule from './features/global-clipboard';
 import handToolOnSpaceModule from './features/hand-tool-on-space';
 import propertiesPanelKeyboardBindingsModule from './features/properties-panel-keyboard-bindings';
 
+import connectorsExtensionModule from 'bpmn-js-connectors-extension';
```

Comment:
Wouldn't that be something that goes via `camunda-bpmn-js`?

Reply:
Some day, maybe, yes.

I'd even argue this should be part of the core. There is no reason not to.